tests/integration.spec.coffee

```diff
@@ -1722,6 +1722,16 @@ describe 'SDK Integration Tests', ->
 						promise = resin.models.os.getDownloadSize('raspberrypi')
 						m.chai.expect(promise).to.eventually.satisfy((n) -> typeof n is 'number')
 
+				describe 'given a specific OS version', ->
+
+					it 'should get a result for ResinOS v1', ->
+						promise = resin.models.os.getDownloadSize('raspberry-pi', '1.24.0')
```

Comment:
Good idea! I've now added a few tests to cover memoization specifically - looks all good.